build/cli/build-cli.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+set -eu
+
+CLI_DIR=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
+
+# mapping of GOOS/GOARCH value to uname value
+declare -A ALL_OS
```

Comment:
the `declare` is not supported in bash 3.x, which means the users using macos will meet an error. this is more of an inconvenience rather than a bug, and can be fixed in a follow-up pr.
i'll suggest to add following code before this line and remove rest of the file as a temporary fix.
```bash
GO111MODULE=on go build -o "${cliBinDir}/alluxioCli-$(uname)-$(uname -m)" cli/main.go
```